nf_core/pipelines/create/template_features.yml

```diff
@@ -401,7 +403,9 @@ test_config:
     - "conf/test_full.config"
     - ".github/workflows/awsfulltest.yml"
     - ".github/workflows/awstest.yml"
-    - ".github/workflows/ci.yml"
+    - ".github/workflows/nf-test.yml"
```

Comment:
when skipping tests, we should also skip `nf-test.config` and the `tests` directory